pkg/monitor/alerts.go

```diff
@@ -84,6 +86,12 @@ func FetchEventIntervalsForAllAlerts(ctx context.Context, restConfig *rest.Confi
 	if err != nil {
 		return nil, err
 	}
+
+	_, err = routeClient.RouteV1().Routes("openshift-monitoring").Get(ctx, "prometheus-k8s", metav1.GetOptions{})
```

Comment:
wouldn't it be easier (and more reliable) to just check for the ns?

Reply:
well, if we'll have openshift-monitoring ns, then it's pretty sure that we'll also have prometheus - changed